assets/javascripts/discourse/initializers/chat-sidebar.js

```diff
@@ -373,13 +372,14 @@ export default {
             }
 
             get actions() {
+              const site = getOwner(this).lookup("site:main");
```

Comment:
`site:main` is deprecated
```suggestion
              const site = getOwner(this).lookup("service:site");
```